app/utils/eventStatus.js

```diff
@@ -36,6 +40,10 @@ const eventStatus = (
       }
       return 'Åpent med påmelding';
     case 'NORMAL':
+      // Check if event has been
+      if (event.startTime < moment()) {
+        return `${registrationCount}/${totalCapacity} påmeldte`;
+      }
```

Comment:
Since the NORMAL and INFINITE cases are the same, you can just put them above each other:
```js
case 'NORMAL':
case 'INFINITE':
...
```